raiden/network/proxies/__init__.py

```diff
@@ -1,6 +1,9 @@
+# isort:skip_file
```

Comment:
why?

Reply:
```
Traceback (most recent call last):
  File "/home/rakan/.pyenv/versions/raiden/bin/raiden", line 11, in <module>
    load_entry_point('raiden', 'console_scripts', 'raiden')()
  File "/home/rakan/Brainbot/raiden/raiden/__main__.py", line 7, in main
    from raiden.ui.cli import run
  File "/home/rakan/Brainbot/raiden/raiden/ui/cli.py", line 19, in <module>
    from raiden.app import App
  File "/home/rakan/Brainbot/raiden/raiden/app.py", line 6, in <module>
    from raiden.network.blockchain_service import BlockChainService
  File "/home/rakan/Brainbot/raiden/raiden/network/blockchain_service.py", line 5, in <module>
    from raiden.network.proxies import (
  File "/home/rakan/Brainbot/raiden/raiden/network/proxies/__init__.py", line 2, in <module>
    from raiden.network.proxies.payment_channel import PaymentChannel  # NOQA
  File "/home/rakan/Brainbot/raiden/raiden/network/proxies/payment_channel.py", line 7, in <module>
    from raiden.network.proxies import TokenNetwork
ImportError: cannot import name 'TokenNetwork' from 'raiden.network.proxies' (/home/rakan/Brainbot/raiden/raiden/network/proxies/__init__.py)
```

Import cycle if the ordering is changed. The alternative is to import `TokenNetwork`:
```
from .token_network import TokenNetwork
```
But we don't really do relative imports AFAIK. so i stuck to the convention and added this instead.